### DISCUSSION TITLE
How could the multiply select should NOT remove the options from dropdown list?

### DISCUSSION THREAD

**Discussion by nazarpelekh** [JedWatson/react-select#5245] [Q&A] [answered]
I'm trying to do a multiply select but I have an issue. When the dropdown is opened and I select an option, it disappears from the dropdown. Example https://codesandbox.io/s/codesandboxer-example-forked-kc6nsp?file=/example.tsx

It is possible to stop removing the option from the dropdown menu?

**Comment by manjushsh** [comment, marked as answer]
Hi @nazarpelekh this can be easily achieved by passing `hideSelectedOptions={false}` to [Select component](https://github.com/JedWatson/react-select/blob/ec3bd2eb7f5b10fa6ce3f9cac9d811a46a74f42a/packages/react-select/src/Select.tsx#L158).

> I'm trying to do a multiply select

as in?

[Code Sandbox](https://codesandbox.io/s/codesandboxer-example-forked-bggw23?file=/example.tsx)

Code:
```
<Select
        defaultValue={[colourOptions[2], colourOptions[3]]}
        isMulti
        name="colors"
        options={colourOptions}
        className="basic-multi-select"
        classNamePrefix="select"
        menuIsOpen
        hideSelectedOptions={false}
      />
```

**Reply by nazarpelekh** [reply]
it works, thanks!
